src/components/Display.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Display(props) {
+  const { value } = props;
+
+  return (
+    <div className="display-container" id="display-container">
+      <p>{ value }</p>
+    </div>
+  );
+}
+
+Display.propTypes = {
+  value: PropTypes.string.isRequired,
+};
```

Comment:
- Set the default value of value here as done in this [document](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values)